cli/version_newer/src/main/scala/scala/scalanative/cli/utils/LinktimePropertyParser.scala

```diff
@@ -0,0 +1,46 @@
+package scala.scalanative.cli.utils
+
+import java.util.regex.Pattern
+import scala.util.Try
+
+object LinktimePropertyParser {
+
+  val ltpPattern = Pattern.compile("(.*?)=(.*)")
+
+  @inline
+  def getLtpPatternException(incorrectPattern: String) =
+    new IllegalArgumentException(
+      s"""|Link-time resolved properties must be of pattern keystring=boolean
+          |\"${incorrectPattern}\" is incorrect.""".stripMargin
+    )
+
+  def toMap(ltpStrings: List[String]): Either[Throwable, Map[String, Any]] = {
```

Comment:
Maybe `parseAll` would be better name for this method? Other methods and fields in this file should be made private